vendor/engines/sanger_sequencing/app/controllers/sanger_sequencing/admin/batches_controller.rb

```diff
@@ -0,0 +1,19 @@
+module SangerSequencing
+
+  module Admin
+
+    class BatchesController < BaseController
+
+      admin_tab :all
+      # layout "two_column"
+      # before_filter { @active_tab = "admin_sanger_sequencing" }
```

Comment:
I haven't quite decided yet. I'll probably add them for some actions, but not others.